src/test/java/com/wikia/webdriver/elements/mercury/old/PortableInfoboxObject.java

```diff
@@ -157,8 +154,8 @@ public PortableInfoboxObject clickGalleryButton(int index) {
     return this;
   }
 
-  public PortableInfoboxObject clickNextImageArrow() {
-    wait.forElementClickable(nextImageArrow).click();
+  public PortableInfoboxObject clickNextImageArrow(int childIndex) {
+    wait.forElementClickable(By.cssSelector(".pi-image:nth-child(" + childIndex + ") .action-next")).click();
```

Comment:
can we extract this By.cssSelector to separate variable so this line is more readable?